scheds/include/scx/compat.bpf.h

```diff
@@ -125,6 +125,11 @@ bool scx_bpf_dispatch_vtime_from_dsq___compat(struct bpf_iter_scx_dsq *it__iter,
 	false;									\
 })
 
+#define scx_bpf_now()								\
```

Comment:
Maybe we could add a little description of what `scx_bpf_now()` does, or even something like "Return a monotonically non-decreasing time in nanoseconds, use `bpf_ktime_get_ns()` if you have high-precision timing requirements" or something similar.